cirq-core/cirq/experiments/readout_confusion_matrix_test.py

```diff
@@ -159,3 +188,80 @@ def test_readout_confusion_matrix_repr_and_equality():
     eq.add_equality_group(a, a)
     eq.add_equality_group(b, b)
     eq.add_equality_group(c, c)
+
+
+def _sample_ghz(
+    n: int, repetitions: int, rng: np.random.Generator = np.random.default_rng()
+) -> np.ndarray:
+    """Sample a GHZ state in the z basis.
+    Args:
+        n: The number of qubits.
+        repetitions: The number of repetitions.
+        rng: The pseudorandom number generator to use.
+    Returns:
+        An array of the measurement outcomes.
+    """
+
+    return np.tile(rng.integers(0, 2, size=repetitions), (n, 1)).T
+
+
+def _add_noise_and_mitigate_ghz(
```

Comment:
there is a significant amount of logic in this method.. tests should be simply and dry (but not too dry). can you simplify the tests?

Reply:
Well, I want to demonstrate not only that the code runs but also that it works on large Pauli operators.

Reply:
if you replace the random part of the function `rng = np.random.default_rng()` with a deterministic one (e.g. `rng = np.random.default_rng(0)`)  then the function becomes determinstic -> not needed. in the test below instead of calling the method just iterate over the values `z_mit, dz_mit, z_raw, dz_raw` from a table.

so just make the test deterministic